src/hct_mis_api/apps/payment/filters.py

```diff
@@ -174,15 +176,37 @@ class Meta:
 class PaymentPlanFilter(FilterSet):
     business_area = CharFilter(field_name="business_area__slug", required=True)
     search = CharFilter(method="search_filter")
-    status = MultipleChoiceFilter(field_name="status", choices=PaymentPlan.Status.choices)
+    status = MultipleChoiceFilter(
+        method="filter_by_status", choices=PaymentPlan.Status.choices + [("ASSIGNED", "Assigned")]
```

Comment:
+ [("ASSIGNED", "Assigned")] ?

Reply:
so it's little hack to filter for TP list all assigned TP to PP. 
![Screenshot 2025-01-08 at 8 52 14 AM](https://github.com/user-attachments/assets/ddc77e14-a1de-466a-9c02-02b795949d01)